### PR TITLE
(SIMP-8410) simp-core Update component versions

### PR DESCRIPTION
**Pull request by lnemsick-simp**
SIMP-8410 #close

### CODE REVIEW

**Comment by lnemsick-simp** [comment]
simp.spec was updated earlier when simp-cli was moved to the master branch in Puppetfile.pinned

**Review by trevor-vaughan — DISMISSED** [review]

**Comment by trevor-vaughan** [comment]
@lnemsick-simp It looks like the EL6 test is probably failing on flipping into FIPS mode since the underlying host can no longer connect with the default ciphers.

The relevant `net-ssh` options appear to be:

```ruby
 pry(main)> Net::SSH::Transport::Algorithms::ALGORITHMS
=> {:host_key=>
  ["ecdsa-sha2-nistp521-cert-v01@openssh.com",
   "ecdsa-sha2-nistp384-cert-v01@openssh.com",
   "ecdsa-sha2-nistp256-cert-v01@openssh.com",
   "ecdsa-sha2-nistp521",
   "ecdsa-sha2-nistp384",
   "ecdsa-sha2-nistp256",
   "ssh-rsa-cert-v01@openssh.com",
   "ssh-rsa-cert-v00@openssh.com",
   "ssh-rsa",
   "ssh-dss"],
 :kex=>
  ["ecdh-sha2-nistp521",
   "ecdh-sha2-nistp384",
   "ecdh-sha2-nistp256",
   "diffie-hellman-group-exchange-sha256",
   "diffie-hellman-group14-sha1",
   "diffie-hellman-group-exchange-sha1",
   "diffie-hellman-group1-sha1"],
 :encryption=>
  ["aes256-ctr",
   "aes192-ctr",
   "aes128-ctr",
   "aes256-cbc",
   "aes192-cbc",
   "aes128-cbc",
   "rijndael-cbc@lysator.liu.se",
   "blowfish-ctr",
   "blowfish-cbc",
   "cast128-ctr",
   "cast128-cbc",
   "3des-ctr",
   "3des-cbc",
   "idea-cbc",
   "none"],
 :hmac=>
  ["hmac-sha2-512-etm@openssh.com",
   "hmac-sha2-256-etm@openssh.com",
   "hmac-sha2-512",
   "hmac-sha2-256",
   "hmac-sha1",
   "hmac-sha2-512-96",
   "hmac-sha2-256-96",
   "hmac-sha1-96",
   "hmac-ripemd160",
   "hmac-ripemd160@openssh.com",
   "hmac-md5",
   "hmac-md5-96",
   "none"],
```

File: spec/acceptance/nodesets/default.yml [hunk 1]
```diff
@@ -56,6 +56,9 @@ CONFIG:
   log_level: verbose
   type:      aio
   vagrant_memsize: 256
+  ssh:
+    keepalive: true
+    keepalive_interval: 1
```
Comment (trevor-vaughan) [inline spec/acceptance/nodesets/default.yml:61]: I *think* this should work

```suggestion
  ssh:
    keepalive: true
    keepalive_interval: 1
    kex:
      - ecdh-sha2-nistp521
      - ecdh-sha2-nistp384
      - ecdh-sha2-nistp256
      - diffie-hellman-group-exchange-sha256
      - diffie-hellman-group14-sha1
      - diffie-hellman-group-exchange-sha1
      - diffie-hellman-group1-sha1
    encryption:
      - aes256-ctr
      - aes192-ctr
      - aes128-ctr
      - aes256-cbc
      - aes192-cbc
      - aes128-cbc
    hmac:
      - hmac-sha2-512
      - hmac-sha2-256
      - hmac-sha1
```

Comment (trevor-vaughan) [inline spec/acceptance/nodesets/default.yml:61]:
```suggestion
  ssh:
    keepalive: true
    keepalive_interval: 1
    host_key:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:host_key].join("\n      - ") %>
    kex:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:kex].join("\n      - ") %>
    encryption:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:encryption].join("\n      - ") %>
    hmac:
      - <%= Net::SSH::Transport::Algorithms::ALGORITHMS[:hmac].join("\n      - ") %>
```

Comment (lnemsick-simp) [reply spec/acceptance/nodesets/default.yml:61]: Acceptance tests are failing at random places with ssh connection failures for a reason unrelated to simp cli.  Should be addressed in a separate ticket.

**Comment by lnemsick-simp** [comment]
simp cli version has bumped again.  Will update when the component is released.

**Comment by lnemsick-simp** [comment]
Discussion about ssh algorithms, etc. will be resolved in PR #690

**Review by trevor-vaughan — APPROVED** [review]

**Pull request merged**